docker/scripts/install-python.sh

```diff
@@ -11,10 +11,26 @@
 # limitations under the License.
 
 
-set -ex
-python3 -m pip install virtualenv numpy
+set -e
+PYTHON_VERSION=${PYTHON_VERSION:-PYTHON3.6}
 DISC_VENV=/opt/venv_disc
 
+function install_python() {
+  if [[ "$PYTHON_VERSION" == "PYTHON3.6" ]]; then
+    apt-get install -y python3.6 python3.6-dev python3-pip
+  elif [[ "$PYTHON_VERSION" == "PYTHON3.8" ]]; then
+    apt-get install -y python3.8 python3.8-dev python3-pip
+    rm -rf /usr/bin/python3
```

Comment:
Nit: Did rm will fully uninstall `/usr/bin/python3` and its related resources?

Reply:
The `python` binary would find the correct related source e.g. `site-packages`, In actually, `/usr/bin/python` or `/usr/bin/python3` is also a soft like usually.